modules/dr-restoring-cluster-state.adoc

```diff
@@ -314,3 +314,29 @@ etcd-ip-10-0-173-171.ec2.internal                2/2     Running     0
 ----
 
 Note that it might take several minutes after completing this procedure for all services to be restored. For example, authentication by using `oc login` might not immediately work until the OAuth server pods are restarted.
+
+[id="dr-scenario-cluster-state-issues_{context}"]
+= Issues and workarounds for restoring persistent storage state
+
+If your {product-title} uses persistent storage of any form, typically a state of the world is stored outside etcd. It could be an ElasticSearch cluster running in a pod or a database running in a statefulset. When restoring from an etcd backup, the status of the workloads in {product-title} is also restored. However, if the etcd snapshot is old, the status might be invalid or outdated.
+
+[IMPORTANT]
+====
+The contents of PVs are never part of the etcd snapshot. When restoring an {product-title} cluster from an etcd snapshot, non-critical workloads might gain access to critical data (or vice-versa).
```

Comment:
- You should spell out PV on first instance here but not later in the same module.

- s/data (or vice-versa)./data, or vice versa.